src/core/probe/bpf/include/common-defs.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __CORE_PROBE_COMMON_DEFS__
+#define __CORE_PROBE_COMMON_DEFS__
+
+#include <vmlinux.h>
+
+/* Global probes configuration; keep in sync with its Rust counterpart in
+ * core::probe::common.
+ */
+struct retis_global_probe_config {
+	u8 comm;
+} __attribute__((packed));
+
+/* Probe configuration; a single entry is used */
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__uint(max_entries, 1);
+	__type(key, u32);
+	__type(value, struct retis_global_probe_config);
+} global_config_map SEC(".maps");
```

Comment:
so you are suggesting reusing the per-probe map for the kernel, right?
I'm not convinced by the fact we'd need to fill it for each probe (sym as key). For the stack trace makes sense as this could be enabled per probe in the future, but for comm, idk, I think of it more like a global on/off switch (I expect more global on-off switches in the future). This would also bloat a bit the data structure (if one considers future additions) and scatter a bit the preparation part (in the probe manager and so forth).
For usdts, were you thinking of something global (one entry)?

Reply:
I guess this doesn't apply anymore